docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1206,6 +1206,24 @@ As the core components use emotion as a styled engine, the props used by emotion
   +</Stepper>
   ```
 
+### styled
```

Comment:
Will be moved under `### @material-ui/core/styled` after https://github.com/mui-org/material-ui/pull/26100 is merged